libs/liblobby/lobby/lobby.lua

```diff
@@ -1364,8 +1378,8 @@ function Lobby:GetUnusedTeamID()
 	local unusedTeamID = 0
 	local takenTeamID = {}
 	for name, data in pairs(self.userBattleStatus) do
-		if data.TeamNumber and not data.isSpectator then
-			local teamID = data.teamNumber
+		local teamID = data.teamNumber
```

Comment:
TeamNumber -> teamNumber? Is this a bug fix?

Reply:
indeed, ZK protocol seems to use TeamNumber, but this is converted to teamNumber at the interface level:
```
	local status = {
		isSpectator   = data.IsSpectator,
		allyNumber    = data.AllyNumber,
		teamNumber    = data.TeamNumber,
		sync          = data.Sync,
	}
```